pkg/object/infos.go

```diff
@@ -4,21 +4,34 @@
 package object
 
 import (
+	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/cli-runtime/pkg/resource"
+	"sigs.k8s.io/kustomize/kyaml/kio/kioutil"
 )
 
 func InfoToUnstructured(info *resource.Info) *unstructured.Unstructured {
 	return info.Object.(*unstructured.Unstructured)
 }
 
-func UnstructuredToInfo(obj *unstructured.Unstructured) *resource.Info {
+func UnstructuredToInfo(obj *unstructured.Unstructured) (*resource.Info, error) {
+	accessor, _ := meta.Accessor(obj)
```

Comment:
Minor nit: we don't need the `meta.Accessor`. `unstructured` has a simple `GetAnnotations` and `SetAnnotations` function. So it would just be `obj.GetAnnotations()` and `obj.SetAnnotations(annots)`.